shared/middleware/helpers/property_bag.rb

```diff
@@ -114,6 +116,12 @@ def set(name, value, ip_address, time = DateTime.now)
       },
     )
     value
+  rescue Aws::DynamoDB::Errors::ValidationException => e
+    if e.message == 'Item size has exceeded the maximum allowed size'
+      too_large
```

Comment:
I'm not sure. Ruby magic? Probably rack. I got TooLarge by copying how NotFound is defined.